### DISCUSSION TITLE
Set Point warm water

### DISCUSSION THREAD

**Discussion by DanielSoftwareDev** [signalkraft/mypyllant-component#194] [Ideas]
Hi,
actually is it not possible to set the warmwater temperature, right? is it easy possible to enable it?
it wouble be greate cause then i can via automation set the temperatur high if i got solar power. thanks for your comments

**Comment by dylan09** [comment]
Could you change the temperature inside the Vaillant app?

Than you also could use service `water_heater.set_temperature`. Give it the entity of your water-heater and the desired set point. I use this to change the setpoint during the day to heat up when sun is shining.

**Reply by DanielSoftwareDev** [reply]
nice i did not found this service under myvaillant. thanks a lot

-CLOSE-

**Comment by Weazle** [comment]
This has worked for me for some time, I had setup an automation to heat my water to a higher temp when my solar panels have overproduction. Also made an automation to have the temp set to 60 for a few hours instead of the legionella program as 70 degrees celsius is not needed with regular use. But for some time now I get an error:

aiohttp.client_exceptions.ClientResponseError: 401, message='Unauthorized', url='https://api.vaillant-group.com/service-connected-control/end-user-app-api/v1/systems/c6c70696-22da-4dba-96b0-f04f6182aaab/tli/domestic-hot-water/255/temperature'

Has anything changed here? I could not find it as issue.
